### PR TITLE
Added selectdropdown verification

### PR DESCRIPTION
**Pull request by zainafzal-tek**
Added Runner2 Class for Parallel Tests
Updated Driver class to support Grid
Updated Driver class to support Parallel Test
Updated POM.xml to support Cucumber reports
Updated Runner class to support additional reports

!! Please DO read Before merging, Before making changes Please merge on local machine

### CODE REVIEW

**Comment by zainafzal-tek** [comment]
I want Rene and Evgeniya to review before merging to avoid any conflicts!!!

File: src/test/resources/features/Account_Activity/AccountActivityNavigation.feature [hunk 1]
```diff
@@ -10,11 +10,14 @@ Feature: As a user I want to be able to navigate in accounts in Accounts Activit
   Scenario: "Account Activity" page displayed
     Then user should see "Zero - Account Activity" page displayed
 
+  @savings_dropdown_default
```
Comment (zainafzal-tek) [inline src/test/resources/features/Account_Activity/AccountActivityNavigation.feature:13]: This is new scenario

File: src/test/java/com/zerobank/utilities/Driver.java [hunk 2]
```diff
@@ -5,46 +5,82 @@
...
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.URL;
 
 public class Driver {
 
     private Driver(){}
 
-    private static WebDriver driver;
+    private static ThreadLocal<WebDriver> driverPool = new ThreadLocal<>();
```
Comment (zainafzal-tek) [inline src/test/java/com/zerobank/utilities/Driver.java:17]: Added parallel execution here

File: src/test/java/com/zerobank/step_definitions/AccountActivity_step_definitions.java [hunk 3]
```diff
@@ -26,5 +26,9 @@ public void transactions_table_should_have_column_names(List<String> expectedHea
         accountActivityPage.verifyHeaders(expectedHeaders);
     }
 
+    @Then("{string} account should be selected")
```
Comment (zainafzal-tek) [inline src/test/java/com/zerobank/step_definitions/AccountActivity_step_definitions.java:29]: Step added here

File: src/test/java/com/zerobank/runners/CucumberRunner2.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package com.zerobank.runners;
```
Comment (zainafzal-tek) [inline src/test/java/com/zerobank/runners/CucumberRunner2.java:1]: Second Runner for parallel testing

File: src/test/java/com/zerobank/runners/CucumberRunner.java [hunk 5]
```diff
@@ -11,13 +11,14 @@
                 "rerun:target/rerun.txt",
                 "json:target/cucumber.json",
                 "pretty",
-                "html:target/cucumber-reports.html"
+                "timeline:target/timeline-report"
```
Comment (zainafzal-tek) [inline src/test/java/com/zerobank/runners/CucumberRunner.java:14]: Replaced Html report with timeline report

File: src/test/java/com/zerobank/pages/AccountActivityPage.java [hunk 6]
```diff
@@ -26,5 +30,13 @@ public void verifyHeaders(List<String> expectedHeaderText){
 
     }
 
+    public void verifySavingsOptionDefault(){
```
Comment (zainafzal-tek) [inline src/test/java/com/zerobank/pages/AccountActivityPage.java:33]: Method for getting first option and getting the  text to assert.

File: pom.xml [hunk 7]
```diff
@@ -30,11 +30,39 @@
                 <version>3.0.0-M5</version>
                 <configuration>
                     <testFailureIgnore>true</testFailureIgnore>
+                    <parallel>classes</parallel>
```
Comment (zainafzal-tek) [inline pom.xml:33]: Parallization edits

File: pom.xml [hunk 8]
```diff
@@ -30,11 +30,39 @@
...
+                    <forkCount>2C</forkCount>
+                    <perCoreThreadCount>false</perCoreThreadCount>
                     <includes>
-                        <include>**/CucumberRunner.java</include>
+                        <include>**/CucumberRunner*.java</include>
                     </includes>
                 </configuration>
             </plugin>
+
+            <plugin>
```
Comment (zainafzal-tek) [inline pom.xml:43]: Reports plugin

**Review by RGallardo — APPROVED** [review]

**Comment by RGallardo** [comment]
All looks great! Nicely done!

**Pull request merged**